_pages/cfp.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+permalink: /call-for-papers/
+title: call for papers
+description:
+nav: true
+nav_order: 1
+---
+
+We welcome researchers working in the field of time series analysis to submit their latest original research work to the NeurIPS 2024 workshop on **Time Series in the Age of Large Models**.
+
+## key information
+
+- Submission link: [TBD](https://openreview.net)
+- Submission deadline: Sep 15, 2024 (11:59 pm AoE)
+- Acceptance notification: Oct 14, 2024
+- Camera ready deadline: Nov 25, 2024
+
+## submission instructions
+
+Submissions should take the form of a short paper of up to **4 pages**. Additional pages containing references and appendices are allowed but the reviewers are _not obliged_ to refer to the appendices when reviewing the paper. Submissions should be made on **[Open Review]()** in a single `.pdf` file using **[this LaTeX style template](../assets/latex/timeseries-workshop-latex-template.zip)**. The review process in _double-blind_, so please ensure that your submission is properly anonymized. Papers that exceed the page limit or have not been properly anonymized will be rejected without review.
+
+All accepted submissions will be accompanied by a poster presentation. A number of selected submissions will be invited for lightning and oral talks. 
+
+**Dual submission policy**: This workshop is **non-archival**; even though all accepted papers will be available on OpenReview and this website, there are no formally-published proceedings. If a paper is currently under review at another venue, it can still be submitted to this workshop. If a paper has previously appeared in a journal, workshop, or conference (including papers accepted to NeurIPS 2024 conference), it should be reasonably extended in order to be accepted at our workshop.
+
+## scope and topics
+
+We invite submissions related to pretrained time series models. The key topics include (but are not limited to):
+
+- Building _foundational_ models for time series data.
+- Analyzing existing pretrained time series models (e.g., explainability and interpretability).
+- Faster and better inference schemes for autoregressive pretrained models.
+- Leveraging pretrained models of other modalities for time series.
+- Multimodal time series models.
+- Large scale datasets, synthetic data generation and data augmentation for time series.
+- Evaluations and real-world applications.
+- Are pretrained models for time series a mirage?
```

Comment:
I feel this can be something that's better for the panel discussion

Reply:
Indeed, but we can also have this as a potential submission topic. Maybe we can reword this here and keep this as the topic of panel discussion. Can we call this "Critiques of foundational models for time series"?

Reply:
"Critiques of foundational models for time series" makes more sense. Modified it myself.